.github/workflows/code-health.yml

```diff
@@ -28,7 +23,39 @@ jobs:
           node-version: 16.x
       - name: Install dependencies
         run: yarn install --check-files
-      - name: build
-        run: npx projen build
-
+      - name: compile
+        run: npx projen compile
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: actionlint
+        run: |
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
+          ./actionlint -color
+        shell: bash
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: Install dependencies
+        run: yarn install --check-files
+      - name: lint
+        run: npx projen eslint
+  dependency-review:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' }}
```

Comment:
does this task depends on the previous tasks? if no then you can move this task to its won workflow and set the workflow to only run on PRs instead of this if

Reply:
> does this task depends on the previous tasks?

no, it doesn't.

> if no then you can move this task to its won workflow and set the workflow to only run on PRs instead of this if

The idea is that all the code-health jobs are grouped in the same worflow